Add build/transfers/index page (#275)
wormhole-foundation/wormhole-docs: build/transfers/index.md

---

# Multichain Transfers

Wormhole transfer products offer multiple asset transfer options to meet developer needs and use cases. These sections include guides to integrate and use Wormhole transfer products to securely and efficiently transfer assets and messages across multiple blockchains.

Use the following links to jump directly to each Wormhole transfer product page or continue on for a feature-based product comparison.

- [**Connect**](/docs/build/transfers/connect/) - a pre-built bridging UI for cross-chain token transfers, requiring minimal setup
- [**Native Token Transfers (NTT)**](/docs/build/transfers/native-token-transfers/) - a mechanism to transfer native tokens cross-chain seamlessly without converting to a wrapped asset
- [**Token Bridge**](/docs/learn/transfers/token-bridge/) - a bridging solution that uses a lock and mint mechanism

> - [**Wormhole Settlement**](/docs/build/transfers/wormhole-docs/build/transfers/settlement/) - an intent protocol suite for fast multichain transfers, optimizing liquidity flows and interoperability without relying on traditional bridging methods

Reviewer: ```suggestion
- [**Settlement**](/docs/build/transfers/wormhole-docs/build/transfers/settlement/) - an intent protocol suite for fast multichain transfers, optimizing liquidity flows and interoperability without relying on traditional bridging methods
```

> - [**Wormhole Settlement**](/docs/build/transfers/wormhole-docs/build/transfers/settlement/) - a next-generation suite of intent protocols enabling fast multichain transfers, optimizing liquidity flows and interoperability without relying on traditional bridging methods

Reviewer: ```suggestion
- [**Settlement**](/docs/build/transfers/wormhole-docs/build/transfers/settlement/) - a next-generation suite of intent protocols enabling fast multichain transfers, optimizing liquidity flows and interoperability without relying on traditional bridging methods
```

Reviewer: This does feel a bit like a marketing line. Maybe we can scale it back a little to focus more on the product, like get rid of "next generation" or something

Reviewer 2: An intent protocol suite for fast multi-chain transfers

> 

Reviewer 2: niceee i like this intro! it's simple and gives a good overview

## Products By Feature

While all of these products handle token transfers, there are additional features to consider when selecting the best fit for your project needs.

<div class="grid cards" markdown>

> -   :octicons-gear-16:{ .lg .middle } **Message Payload**

Reviewer: If the title for NTT is Native Assets maybe this one can just be Wrapped Assets? Wdyt? I was just thrown off by "message payloads" for some reason when I first saw it, so if not Wrapped Assets, maybe something else?

Author: It's actually key that you can send a message or data payload along with the token using Token Bridge. This is what allows you to create non-fungible tokens, social platform interactions, etc. This feature mention is not about wrapped tokens versus native.

> -   :octicons-gear-16:{ .lg .middle } **Message Payload**

Reviewer: Not sure about the gear icon for all of these. @ilariae I know you went through and generalized the icons, did you by chance document anything related to the usage of icons?

Reviewer 2: yep here --> https://docs.google.com/spreadsheets/d/1JxpVa9xOMvXnhBpAx6zBG0wP3Bej0X7CIVwO4YzKFVw/edit?usp=sharing

Author: This doesn't have the categories for the new IA but I went with `file-code` Happy to change if something else is preferred.

    ---

>     Send message payloads for social platforms and NFTs with Token Bridge's lock and mint wrapped token mechanism.

Reviewer: ```suggestion
    Send message payloads for social platforms, NFTs, and more with Token Bridge's lock-and-mint wrapped token mechanism.
```

To generalize it a bit 🤷‍♀️

>     [:custom-arrow: Discover Token Bridge](/docs/learn/transfers/token-bridge/)

Reviewer: ```suggestion
    [:custom-arrow: Discover Token Bridge](/docs/build/transfers/token-bridge/)
```

>     [:custom-arrow: Discover Token Bridge](/docs/learn/transfers/token-bridge/){target=\_blank}

Reviewer: ```suggestion
    [:custom-arrow: Discover Token Bridge](/docs/learn/transfers/token-bridge/)
```

-   :octicons-gear-16:{ .lg .middle } **Native Assets**

    ---

>     Eliminate the need for wrapped tokens and issue native multichain assets with Native Token Transfer (NTT).

Reviewer: ```suggestion
    Eliminate the need for wrapped tokens and issue native multichain assets with Native Token Transfers (NTT).
```

>     [:custom-arrow: Discover Native Token Transfer](/docs/build/transfers/native-token-transfers/)

Reviewer: ```suggestion
    [:custom-arrow: Discover Native Token Transfers](/docs/build/transfers/native-token-transfers/)
```

-   :octicons-file-code-16:{ .lg .middle } **Speed at Scale**

    ---

    Unleash institutional-scale digital asset settlement with Wormhole Settlement's intent-based asset transfers and liquidity layer.

>     [:custom-arrow: Discover Settlement](/docs/build/transfers/wormhole-docs/build/transfers/settlement/)

Reviewer: ```suggestion
    [:custom-arrow: Discover Settlement](/docs/build/transfers/settlement/)
```

> -   :octicons-gear-16:{ .lg .middle } **Plug and Play UI**

Reviewer: ```suggestion
-   :octicons-gear-16:{ .lg .middle } **Plug-and-Play UI**
```

    ---

>     Add Wormhole token transfer UI to your dApp, no smart contract development required, with Connect.

Reviewer: ```suggestion
    Add Wormhole's token transfer UI to your dApp, no smart contract development required, with Connect.
```

Not sure if this is what this needs, but it just sounds a little off

Reviewer 2: maybe we should call it bridge UI

    [:custom-arrow: Discover Connect](/docs/build/transfers/connect/)

</div>

> 

Reviewer 2: these features are spot on 🚀🚀🚀 i really like this

## Additional Resources

<div class="grid cards" markdown>

-   :octicons-tools-16:{ .lg .middle } **Product Comparison**

    ---

>     Compare Wormhole’s cross-chain solutions for bridging, native transfers, data queries, and governance to enable seamless blockchain interoperability.

Reviewer: ```suggestion
    Compare Wormhole's cross-chain solutions for bridging, native transfers, data queries, and governance to enable seamless blockchain interoperability.
```

>     [:custom-arrow: Compare Products](/docs/build/start-building/products/){target=\_blank}

Reviewer: ```suggestion
    [:custom-arrow: Compare Products](/docs/build/start-building/products/)
```
